packages/tools/devtools/devtools-view/src/components/data-visualization/FluidHandleView.tsx

```diff
@@ -87,5 +88,27 @@ export function FluidHandleView(props: FluidHandleViewProps): React.ReactElement
 		return <TreeItem header={header} />;
 	}
 
-	return <TreeDataView containerKey={containerKey} label={label} node={visualTree} />;
+	//Button to inlcude in header to copy fluid handle
+	const button = (
+		<Tooltip content={`Click to copy handle`} relationship={"label"}>
+			<Button
+				onClick={async (e): Promise<void> => {
+					e.preventDefault();
+					return navigator.clipboard.writeText(fluidObjectId);
+				}}
+				icon={<ClipboardPaste16Regular />}
+				size="small"
+			></Button>
+		</Tooltip>
+	);
+
+	const header2 = (
+		<TreeHeader label={label} inlineValue={button} nodeTypeMetadata={"FluidHandle"} />
+	);
+
+	return (
+		<TreeItem header={header2}>
+			<TreeDataView containerKey={containerKey} label={"FluidObject"} node={visualTree} />
```

Comment:
Nitpick: What do you think about "value" instead of "FluidObject"? Or "data"?

Reply:
I think data would go well, it woudld then be consistent with how sharedCell looks with how it shows its values. Will change it